### PR TITLE
Component/feature banner

### PR DESCRIPTION
**Pull request by SamGregory1**

### CODE REVIEW

File: vendor/assets/stylesheets/ustyle/components/_hero.scss [hunk 1]
```diff
@@ -24,6 +24,10 @@ $hero-text-color: $c-navy !default;
   @extend %container;
   max-width: 800px;
   padding: 2.5em 15px;
+
+  @include respond-to(desktop) {
```
Comment (annez) [inline vendor/assets/stylesheets/ustyle/components/_hero.scss]: This is already in the grid? Can you confirm why this is being done?

File: vendor/assets/stylesheets/ustyle/components/_hero.scss [hunk 2]
```diff
@@ -32,7 +36,7 @@ $hero-text-color: $c-navy !default;
 
   @include respond-to(tablet) {
     margin-top: 0;
-    font-size: 2.25em;
+    font-size: 2.37em;
```
Comment (annez) [inline vendor/assets/stylesheets/ustyle/components/_hero.scss]: Don't do this, keep it default, don't match what is on site, it's wrong.

File: vendor/assets/stylesheets/ustyle/components/_hero.scss [hunk 3]
```diff
@@ -44,6 +48,8 @@ $hero-text-color: $c-navy !default;
 .us-hero-description {
   @extend %reset-box-model;
   display: none;
+  padding-top: 5px;
+  color: $c-typegrey;
```
Comment (annez) [inline vendor/assets/stylesheets/ustyle/components/_hero.scss]: What colour is it currently?

Comment (SamGregory1) [reply vendor/assets/stylesheets/ustyle/components/_hero.scss]: Depends where you look. On Bad Credit (And other places I imagine), it's inheriting the body which is c-navy (#1c1f4e) and others use c-typegrey (#2f353a). I used the colour that was on Bad Credit.

File: vendor/assets/stylesheets/ustyle/components/_hero.scss (hunk 1 above)
Comment (SamGregory1) [reply vendor/assets/stylesheets/ustyle/components/_hero.scss]: Had some inheritance issues previously. I have removed this now as the issue is no longer prevalent.

**Pull request merged**